docs/11-dev-guide.md

```diff
@@ -39,19 +42,25 @@ git checkout --track origin/main
 ```
 
 If you're working with the Sleeper CLI, you can use `sleeper builder` to get a shell inside a Docker container with
-the dependencies pre-installed. You'll need to clone the Git repository, and this will be persisted between executions
-of `sleeper builder`. Use the commands below:
+the dependencies pre-installed. **Note that due to a bug in the GitHub container repository you need to build the
+builder image from scratch**, so it's much easer to use Nix. If you do want to build the builder image from scratch, see the [Sleeper CLI](11-dev-guide.md#sleeper-cli) section below
+
+As part of the EC2 setup it will automatically clone the Sleeper Git repository to the `sleeper` folder in the builder image home directory:
```

Comment:
It's not clear which EC2 this is referring to. The environment EC2 is explained elsewhere, so we probably don't need to duplicate that here?